README.md

```diff
@@ -22,6 +23,7 @@ Reference Implementation (IRI).
                    from downloading from
                    [latest JAVA source](https://github.com/chenwei-tw/iri/tree/feat/new_pow_interface).
     - ``BUILD_COMPAT``: build extra cCurl compatible interface.
+    - ``BUILD_FPGA_ACCEL``: build the dcurl interface in communication with the IOTA hardware accelerator on the Cyclone V FPGA board, e.g. DE10-nano board and Arrow SoCKit board. 
```

Comment:
Change the statement to "build the interface interacting with the Cyclone V FPGA based accelerator. Verified on DE10-nano board and Arrow SoCKit board."